docs/library/Result-Handling.md

```diff
@@ -0,0 +1,58 @@
+# Result handling
+
+The search functions of the Nominatim API always return a result object that
```

Comment:
A lot of 'that'.
return a result object containing ...
results and utility functions which allow...